codebuild/codebuild_get_changed_files/codebuild_get_changed_files.yml

```diff
@@ -0,0 +1,21 @@
+version: 0.2
+
+env:
+  shell: bash
+
+phases:
+  build:
+    commands:
+      - echo '[INFO] Reading artifacts...'
+      - pwd
+      - ls
+      - cd $TERRAFORM_DIR
+      - ls
+      - ./github_rest_get_pr_changed_files_list.sh $REPO_NAME /opt/changed_files.json
```

Comment:
I don't think we want to do that here. There's already a module for it here: https://github.com/alphagov/cyber-security-shared-terraform-modules/blob/master/codebuild/codebuild_git_diff/code_build_git_diff.yml

We just want to do this once and pass this around as an artifact.